cmake/pcl_find_cuda.cmake

```diff
@@ -24,16 +24,12 @@ if(CUDA_FOUND)
 
   # Find a complete list for CUDA compute capabilities at http://developer.nvidia.com/cuda-gpus
 
-  if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "10.0")
+  if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "11.0")
```

Comment:
Could you post a URL or even better add comment on where to check the appropriate `__cuda_arch_bin` for each toolkit version?

Reply:
I have added the comment.